src/components/ClientSignUp.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+import Auth from "../modules/authentication";
+import { StyleSheet } from "react-native";
+import { useDispatch } from "react-redux";
+import AsyncStorage from "@react-native-community/async-storage";
+import {
+  Container,
+  Button,
+  Content,
+  Form,
+  Item,
+  Input,
+  Label,
+  Text,
+} from "native-base";
+
+const ClientSignUp = (props) => {
+  const dispatch = useDispatch();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [passwordConfirmation, setPasswordConfirmation] = useState("");
+  const [company, setCompany] = useState("");
+  const [companyUrl, setCompanyUrl] = useState("");
+  const [message, setMessage] = useState();
+  const auth = new Auth({ host: "https://develup-2020.herokuapp.com/api" });
+
+  const storage = AsyncStorage;
+
+  const signUpHandler = async () => {
+    try {
+      let response = await auth.signUp({
+        email: email,
+        password: password,
+        password_confirmation: passwordConfirmation,
+        company_name: company,
+        company_url: companyUrl,
+        role: "client",
+      });
+      await storage.setItem("auth-storage", JSON.stringify(response.headers));
+      dispatch({
+        type: "SIGNUP",
+        payload: {
+          authenticated: true,
+          currentUser: response.data.data,
+        },
+      });
+      props.navigation.navigate("clientPage", {
+        clientSignUpMessage: `You are logged in with: ${response.data.data.company_name}!`,
+      });
+    } catch (error) {
+      let errorMessage = error.response.data.errors.full_messages;
+      setMessage(errorMessage);
+    }
+  };
+
+  return (
+    <Container>
+      <Content testID="mainContainer">
+        <Form>
+          <Item floatingLabel>
+            <Label testID="emailInput">Email</Label>
+            <Input
+              testID="emailField"
+              onChangeText={(text) => setEmail(text)}
+            />
+          </Item>
```

Comment:
change the name's of testID, r.61 should be emailLabel and r.63 should be emailInput. Make sure to change in the test